tests/data_science/test_grafana_data_registration.py

```diff
@@ -49,15 +49,15 @@ def setup_method(self):
 
     @pytest.fixture
     def _run_id(self, run):
-        return str(run.id)
+        return str(run.readable_id)
 
     @pytest.fixture
     def _config_id(self, simulation_configuration):
-        return str(simulation_configuration.id)
+        return str(simulation_configuration.readable_id)
 
     @pytest.fixture
     def _multi_config(self, simulation_configuration):
-        return str(simulation_configuration)
+        return str(simulation_configuration.readable_id)
 
```

Comment:
This didn't use the id previously, should it now use the readable id?

Reply:
When calling str() on simulation_configuration, the ID will automatically be used to generate the str, I believe. So when we are now switching to readable ID, I have to specify that directly.

Reply:
Ah okay, thank you for the clarification!